cluster-provision/images/vm-image-builder/customize-image.sh

```diff
@@ -40,6 +55,8 @@ virt-install \
   --network default \
   --import
 
+wait_for_vm_state "shutoff"
```

Comment:
I think this should be optional in order to support local environments in case one would want to have live console to the provisioned VM.

Reply:
It works well in local environments, meaning that the only impact is that the script is be executed just once after the `virt-install` command completes but it does not modify flow in any other way. Initially I planned to hide the console output, but I thought that it added value for local runs/debugging, so I left it intact.
When running in a CI, this function will make sure that the execution does not continue until `virt-install` has completed executing.

Reply:
Since we use this function only here, I think its better to not accept arguments and put `shutoff` arg as part of the function and rename it to something like `wait_for_vm_shutoff`